signals/helpers.py

```diff
@@ -0,0 +1,16 @@
+import os.path
+import glob
+
+
+def recursively_find_parent_containing_file(current_path, search_files):
```

Comment:
I have no solution for how we know when to stop it appropriately though :smiling_imp: